### ISSUE TITLE
[Accessibility] An error pops up when setting a value for 'AccessibleName' property of ComboBox control

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#4408]
* .NET Core Version: 
.NET SDK 6.0.100-alpha.1.20623.10

* Have you experienced this same bug with .NET Framework?: 
Yes

**More Info:**

1. This is a regression issue, it cannot reproduce on .NET SDK 6.0.100-alpha.1.20617.6.
2. Step 2 is necessary, if don't custom comBoBox's AccessibleName property, this issue cannot repro.
3. This issue cannot repro when setting ComBobox's dropDownStyle property as Simple.

**Minimal repro:**

1. Create a .NET Core Winforms application, add a ComboBox with items or without items(Duplicate items are not necessary).
2. Set a string to 'AccessibleName' property for ComboBox(eg: CB1). --- It is necessary step
3. Build and run this project.
4. Open Accessibility Insights tool, and focus on the ComboBox's list part.

Or

1. Open the attached [SampleApp.zip](https://github.com/dotnet/winforms/files/5737802/SampleApp.zip) application in VS, then build and run it.
2. Open Accessibility Insights tool, and focus on the ComboBox's list part.

**Expected behavior:**
No error pops up.
![image](https://user-images.githubusercontent.com/45864649/103051893-c7a72200-45d2-11eb-9ed3-5cba9701ec36.png)

**Actual result:**
Accessibility Insights detects an error for ComboBox's List part.
![image](https://user-images.githubusercontent.com/45864649/103051116-fe7c3880-45d0-11eb-8c94-9b063d193dfa.png)

**Comment by merriemcgaw** [comment]
Let's prioritize this very high

**Comment by vladimir-krestov** [comment]
I couldn't reproduce this issue using `6.0.0-preview.2.21077.7` sdk
@Jessie-Zhang01, could you please recheck if it still repro?

**Comment by vladimir-krestov** [comment]
First thoughts:
1) If ComboBox AccessibleName is default:
     - Inspect tree:
![image](https://user-images.githubusercontent.com/49272759/106417541-5ece8780-6465-11eb-85b1-ebc644f6efde.png)
     - AI result:
![image](https://user-images.githubusercontent.com/49272759/106417405-0eefc080-6465-11eb-8479-41c83996fa2b.png)

     It looks unexpected because we can't change the list name externally. It should have some default AccessibleName.
2)  If ComboBox AccessibleName is custom (this Issue subject):
     - Inspect tree:
![image](https://user-images.githubusercontent.com/49272759/106418014-5fb3e900-6466-11eb-85c9-44d026a4b131.png)
     - AI result:
![image](https://user-images.githubusercontent.com/49272759/106418093-8eca5a80-6466-11eb-9abf-1e14f88858ce.png)

     In my opinion, the accessibility tree looks good. We have different items that belong to the ComboBox (the drop-down list and the edit field) and that have **different control types**. So a user could have differed them using Narrator if Narrator had worked well. Narrator should announce: `"CB1" edit, "Close" button, "CB1" list`, but it works absolutely incorrectly in `Scan` mode:
     - Narrator: 
     ![qHvWXdSUwL](https://user-images.githubusercontent.com/49272759/106419463-93444280-6469-11eb-9e2e-9c2f211c9165.gif)
3) Okay, I can change the default `AccessibleName` of the drop-down list to `"DropDown"`:
     - Inspect tree:
![image](https://user-images.githubusercontent.com/49272759/106419864-76f4d580-646a-11eb-8611-4815727ba273.png)
     - AI result:
![image](https://user-images.githubusercontent.com/49272759/106419967-bf13f800-646a-11eb-82d0-86cf483eb334.png)

     So, the result is the same. But there are no items. All children of the ComboBox have different names and control types. I thought, that we have the incorrect tree and we create some extra invisible item, but it seems wrong because I used AccessibleName as `GetHashCode() + a static-different variable`. This AccessibleName always is different for every call. But the AI bug still reproduces.

     We have a UIA accessibility tree for `ComboBox AccessibleObject`, it was set-up and tested before and should work correctly. Anyway, .Net Framework ComboBox uses the native implementation but it works the same. All points above are applicable for .Net Framework too. In my opinion, this is a bug of `AccessibilityInsights` that sees some non-existent siblings.

**Comment by vladimir-krestov** [comment]
There is this bug even if a ComboBox and its children have different accessibility names:
![image](https://user-images.githubusercontent.com/49272759/106856294-4c578680-66cf-11eb-9289-f35d2ca4a3b9.png)

**Comment by vladimir-krestov** [comment]
Closed this bug as "By Design" because Winforms UI and UIA tree work correctly.
The issue is that AI is trying to run a test on a UI element which no longer exists (the tested drop-down list is already collapsed) - [see comment](https://github.com/microsoft/accessibility-insights-windows/issues/1041#issuecomment-775554846)

Moreover, there is a correct way of testing a ComboBox drop-down list - using the timer in AI:
- run the timer in AI
- expand the tested ComboBox
- just hover the dropped list by mouse without a click when the timer is working

![Ibqm2Bbw7q](https://user-images.githubusercontent.com/49272759/107552870-1c374880-6be5-11eb-99af-9c63f87fe7c4.gif)
